modules/go.work.sum

```diff
@@ -0,0 +1,499 @@
+bitbucket.org/creachadair/shell v0.0.6 h1:reJflDbKqnlnqb4Oo2pQ1/BqmY/eCWcNGHrIUO8qIzc=
```

Comment:
You might want to run `git rm --cached ...` on this file since it was added to the `.gitignore`.

Reply:
Cleaned up the files left behind